backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestEvent.java

```diff
@@ -3,49 +3,110 @@
 import javax.persistence.AttributeOverride;
 import javax.persistence.Column;
 import javax.persistence.Entity;
-import javax.persistence.OneToOne;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
 
+import gov.cdc.usds.simplereport.db.model.auxiliary.AskOnEntrySurvey;
+import gov.cdc.usds.simplereport.db.model.auxiliary.TestCorrectionStatus;
 import org.hibernate.annotations.Immutable;
 import org.hibernate.annotations.Type;
 
 import gov.cdc.usds.simplereport.db.model.auxiliary.TestResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Date;
+import java.util.UUID;
 
 @Entity
 @Immutable
 @AttributeOverride(name = "result", column = @Column(nullable = false))
 public class TestEvent extends BaseTestInfo {
+	private static final Logger LOG = LoggerFactory.getLogger(TestEvent.class);
 
 	@Column
 	@Type(type = "jsonb")
 	private Person patientData;
+
 	@Column
 	@Type(type = "jsonb")
 	private Provider providerData;
-	@OneToOne(mappedBy = "testEvent")
+
+	@Column
+	@Type(type = "jsonb")
+	private AskOnEntrySurvey patientAnswersData;
+
+	@ManyToOne(optional = false)
+	@JoinColumn(name="test_order_id")
 	private TestOrder order;
 
+	@Column(columnDefinition = "uuid")
+	private UUID priorCorrectedTestEventId;	// used to chain events
+
 	public TestEvent() {}
 
-	public TestEvent(TestResult result, DeviceType deviceType, Person patient, Facility facility) {
+	public TestEvent(TestResult result, DeviceType deviceType, Person patient, Facility facility, TestOrder order) {
```

Comment:
This begins to seem like a constructor that should be eliminated, though I guess the short constructor goes through it anyway....